### ISSUE TITLE
Create options for tournaments not to be listed in search engine

### ISSUE THREAD

**Issue by andreesc15** [TabbycatDebate/tabbycat#1724]
There has been several concerns of debaters that they (and their debating history) can be tracked down by googling their name (e.g. Name + "Tab"), and these individuals / institutions have expressed their concerns on their digital footprint (e.g. being tracked by future employers, etc.)

Was wondering if it can be added into list of possible configurations (or when it's deployed), to set whether certain tournament's site wants to be listed in search engine or not? Perhaps by adding `noindex` meta tag (am not sure how Tabbycat Page are generated, but it's just a random idea in my head).

**Comment by philipbelesky** [comment]
This seems sensible as a way of augmenting our existing anonymity options - and perhaps making them less necessary. 

Using a robots.txt value tied to a preference would be easy enough to add. 

My thought would be that you would still want the tournament homepage indexed regardless so that you could google "University IV tab" and get results but not "Jane Smith tab".

**Comment by andreesc15** [comment]
Yup, I think making sure these people's name are not indexed would definitely help them anonymize themselves.

I can think of several scenarios where the entire tab site are not indexed, though it would be extremely rare. But should it be possible, giving the control to do so if the adjcore/committee/tab wishes to do so would be great.

**Comment by czlee** [comment]
Sounds good to me (albeit symptomatic of a regrettable phenomenon). All of the private URL pages are [already (hopefully) marked with `noindex`](https://github.com/TabbycatDebate/tabbycat/search?q=noindex), so another implementation idea would be include that in [base.html](https://github.com/TabbycatDebate/tabbycat/blob/develop/tabbycat/templates/base.html), switched by a preference (using `{% if pref.do_not_index %}` once it's set up in [preferences.py](https://github.com/TabbycatDebate/tabbycat/blob/develop/tabbycat/options/preferences.py), can set it up as a `ChoicePreference` to have multiple levels of noindexing).
